src/snackbar/snackbar.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function MaterialSnackbar(element) {
+  'use strict';
+
+  this.element_ = element;
+  this.active = false;
+  this.init();
+
+}
+
+MaterialSnackbar.prototype.defaults = {
+  timeout: 8000
+};
+
+MaterialSnackbar.prototype.cssClasses = {
+  snackbar: 'mdl-snackbar',
+  message: 'mdl-snackbar__text',
+  action: 'mdl-snackbar__action',
+  activeSnackbar: 'is-active'
+};
+
+MaterialSnackbar.prototype.queuedNotifications = [];
+
+MaterialSnackbar.prototype.setMessage = function(text) {
+  'use strict';
```

Comment:
I was following the same style being used in the rest of the components. I'd rather not deter from that unless we refactor the existing components as well (which I'd be for.) These should all be consistent.